setup.py

```diff
@@ -83,6 +83,7 @@ def run(self):
         'apache-airflow>=1.10.0',
         # FAB is pulled in from Airflow
         'jwcrypto~=0.6.0',
+        'requests',
```

Comment:
not sure why but the main branch started failing w/o adding this package to our deps even though previously the build was passing w/o it

Reply:
Ah, airflow 2.1.0 didn't pull this in explicitly anmore, but prior versions did

Reply:
That said -- what was the error? I don't think anything in this library makes any HTTP requests!

Reply:
See errors on this build. https://app.circleci.com/pipelines/github/astronomer/astronomer-fab-securitymanager/52/workflows/a3d4d229-a200-491f-ae77-c0532d4175fd/jobs/63

The current_user_backend.py file pull in requests for its .auth subpackage.

from requests.auth import AuthBase